app/src/main/java/com/ucmobiledevelopment/freeways/dto/IncidentListDTO.kt

```diff
@@ -4,5 +4,5 @@ data class IncidentListDTO(
     val Count: Int = 0,
     val Message: String = "",
     val Results: List<List<IncidentDTO>>,
-    val SearchCriteria: String = ""
+    val SearchCriteria: String = "",
```

Comment:
Trailing commas, while not required, are an easy way to reduce future technical debt: https://kotlinlang.org/docs/coding-conventions.html#trailing-commas